e2e/tests/workspace/workspace-ui.spec.ts

```diff
@@ -27,36 +29,41 @@ describe('Workspace ui tests', () => {
         width = size.width; // Initialize width and height with first card element's size, compare with rest cards
         height = size.height;
       } else {
+        expect(size.height).toBeGreaterThan(0);
+        expect(size.width).toBeGreaterThan(0);
         expect(size.height).toEqual(height);
         expect(size.width).toEqual(width);
       }
     }
-  });
-
-  test('Check Workspace cards on the Workspaces page', async () => {
```

Comment:
Combined `'Check Workspace cards on the Workspaces page'` and `'Workspace cards'` tests. One less login step.